### ISSUE TITLE
Accuracy will not imcrease during training

### ISSUE THREAD

**Issue by GesilaA** [supercrazysam/Fractional_MaxPooling#4]
I use keras==2.3.1 & tensorflow-gpu==1.14.0
I modified K.set_image_dim_ordering('tf') to K.image_data_format()='channels_last' in 'cifar_test.py' & 'Fractional_MAXPOOL.py'

There is the terminal output:
Epoch 2/30
70/69 [==============================] - 163s 2s/step - loss: 2.3067 - accuracy: 0.1051 - val_loss: 2.3024 - val_accuracy: 0.1001
Epoch 3/30
70/69 [==============================] - 163s 2s/step - loss: 2.3138 - accuracy: 0.1061 - val_loss: 2.3020 - val_accuracy: 0.1201
Epoch 4/30
70/69 [==============================] - 163s 2s/step - loss: 2.3038 - accuracy: 0.0997 - val_loss: 2.3026 - val_accuracy: 0.0999
Epoch 5/30
70/69 [==============================] - 163s 2s/step - loss: 2.3026 - accuracy: 0.1010 - val_loss: 2.3021 - val_accuracy: 0.1000
Epoch 6/30
70/69 [==============================] - 163s 2s/step - loss: 2.3051 - accuracy: 0.1026 - val_loss: 2.3027 - val_accuracy: 0.0999
Epoch 7/30
70/69 [==============================] - 163s 2s/step - loss: 2.4733 - accuracy: 0.1087 - val_loss: 2.2949 - val_accuracy: 0.1220
Epoch 8/30
70/69 [==============================] - 163s 2s/step - loss: 2.4130 - accuracy: 0.1140 - val_loss: 2.3005 - val_accuracy: 0.1104
Epoch 9/30
70/69 [==============================] - 162s 2s/step - loss: 2.2995 - accuracy: 0.1059 - val_loss: 2.2995 - val_accuracy: 0.1760
Epoch 10/30
70/69 [==============================] - 162s 2s/step - loss: 2.3125 - accuracy: 0.1122 - val_loss: 2.3078 - val_accuracy: 0.0999
